### DISCUSSION TITLE
請問未來還會支援 LTP server 嗎

### DISCUSSION THREAD

**Discussion by kanasimi** [HIT-SCIR/ltp#574] [Q&A]
這次 4.2 大改版似乎沒發現 LTP server，不曉得未來會支援嗎？

題外話。需要採用server是因為沒有好的JavaScript接口。採用JavaScript配合python的話每次查詢都要重頭執行程式，初始化速度太慢，只好採用server的形式減少初始化時間。

**Comment by geasyheart** [comment]
使用tornado或者fastapi搭建个server不就好啦

**Reply by kanasimi** [reply]
之前有[server.py](https://github.com/HIT-SCIR/ltp/blob/master/tools/server.py)這支程式所以很好改。不曉得[以後會附上嗎](https://github.com/HIT-SCIR/ltp/blob/d1da0b8c6dc0270f077d2e83c511031083ad2aab/python/interface/docs/quickstart.rst#ltp-server)？

**Comment by geasyheart** [comment]
emm，我不是官方人员，但是作为开发者来讲，你可以参考之前官方例子重新搭建一个，也可以自己技术选型不管http或者rpc等等

**Reply by kanasimi** [reply]
因為LTP有[線上展示](http://ltp.ai/demo.html)，我想應該公開這部分server端的原始碼就夠了...
